modules/doc/content/application_development/profiling.md

```diff
@@ -47,6 +47,22 @@ This will use the filename base you pass and append a suffix of the form
 rank/process. The above example would generate files `run1_0.prof`,
 `run1_1.prof`, `run1_2.prof`, ..., `run1_31.prof`.
 
+### Heap Profiling
+
+Similarly, you can do a heap profiling like this:
+
+```
+HEAP_PROFILE_INUSE_INTERVAL=104857600 MOOSE_HEAP_BASE=run1_ mpiexec -n 32 ./your-app_oprof -i input_file.i
+```
```

Comment:
Maybe we are missing an empty line bellow? Check this out; https://mooseframework.inl.gov/docs/PRs/16224/site/application_development/profiling.html.

Reply:
Thanks. Good catch. I will fix that